OtripleS.Web.Api/Models/Gender.cs

```diff
@@ -7,4 +7,18 @@ public enum Gender
         Other,
         Unknown
     }
-}
+
+    public static class GenderHelper
+    {
+        public static Gender StringToGenderConverter(string gender)
+        {
+            return gender.ToLower() switch
+            {
+                "female" => Gender.Female,
+                "male" => Gender.Male,
+                "other" => Gender.Other,
+                _ => Gender.Unknown
+            };
```

Comment:
API controller does that for your out of the box, you don't really need that here.